### PR TITLE
Blog Submission

### PR DESCRIPTION
**Pull request by husky-parul**
Contributed blog post for sustainability week. 

@rootfs @leonardpahlke

### CODE REVIEW

**Comment by husky-parul** [comment]
The preview does not have the link to this PR blog. Is this expected behavior?

image

File: website/content/blog/Embracing-Edge-Computing-with-Kepler.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Embracing Edge Computing with Kepler: A Dive into Remote Monitoring, Centralized Dashboarding, and Visualization
```
Comment (leonardpahlke) [inline website/content/blog/Embracing-Edge-Computing-with-Kepler.md:1]:
```suggestion
---
title: Embracing Edge Computing with Kepler: A Dive into Remote Monitoring, Centralized Dashboarding, and Visualization
description: Kepler enables remote monitoring, centralized dashboarding, and visualization of energy consumption, revolutionizing energy observability.
slug: 2023-embracing-edge-computing-with-kepler
weight: 1
date:   2023-10-09 10:00:00 +0000
author: Parul Singh
---

```

The header is missing. Next to the title, we also need to add a description. I added a description, but please use any description you think works best!

File: website/content/blog/Embracing-Edge-Computing-with-Kepler.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+![Alt text](../../static/images/blogs/2023-09-cloud-native-sustainability-week/KEPLER-OTEL.png)
+
+## Kepler’s Architectural Brilliance
+
+Kepler’s architecture captures pod/container/processes level metrics efficiently. The OTEL collector acts as the intermediary, processing and routing data to destinations. Leveraging Prometheus as a backend, coupled with Grafana's prowess, transforms raw data into meaningful visualizations. This synergy offers a seamless experience, enabling users to comprehend complex data effortlessly.
+
+In conclusion, Kepler together with OpenTelemetry and its focus on edge computing mark a significant leap in the power monitoring landscape.
+
+Reference: Hybrid Cloud Patterns
```
Comment (leonardpahlke) [inline website/content/blog/Embracing-Edge-Computing-with-Kepler.md:25]: Are we missing a link here?

File: website/content/blog/Embracing-Edge-Computing-with-Kepler.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## The Power of Centralized Dashboarding
+
+Centralized dashboarding emerges as a cornerstone of this paradigm shift. With edge devices operational in various locations, having a centralized dashboard becomes indispensable. It offers a unified view of all the monitored parameters, ensuring seamless monitoring and identifying major power consuming workloads and/or devices. Centralized dashboarding ensures that insights into equipment and workload power consumption are readily accessible, enabling timely decision-making and proactive issue resolution.
+
+## The Power of OpenTelemetry
+
+Embracing OpenTelemetry isn't just a trend; it's a necessity. This open standard for observability ensures vendor-neutral instrumentation, collection, and export of telemetry data. OpenTelemetry's flexibility, consistency, and growing ecosystem, coupled with support from major observability vendors, position it as the go-to choice for modern applications.
+
+![Alt text](../../static/images/blogs/2023-09-cloud-native-sustainability-week/KEPLER-OTEL.png)
```
Comment (leonardpahlke) [inline website/content/blog/Embracing-Edge-Computing-with-Kepler.md:17]: We may want to add some short explanatory text next to the diagram. WDYT?

File: website/content/blog/Embracing-Edge-Computing-with-Kepler.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Kepler’s Architectural Brilliance
+
+Kepler’s architecture captures pod/container/processes level metrics efficiently. The OTEL collector acts as the intermediary, processing and routing data to destinations. Leveraging Prometheus as a backend, coupled with Grafana's prowess, transforms raw data into meaningful visualizations. This synergy offers a seamless experience, enabling users to comprehend complex data effortlessly.
+
+In conclusion, Kepler together with OpenTelemetry and its focus on edge computing mark a significant leap in the power monitoring landscape.
+
+Reference: Hybrid Cloud Patterns
+
+*Note: Follow [link](https://github.com/husky-parul/otel-observability) for the deployment steps. Stay tuned for more updates on [Kepler OTEL SDK instrumentation](https://github.com/sustainable-computing-io/kepler/issues/659).*
```
Comment (leonardpahlke) [inline website/content/blog/Embracing-Edge-Computing-with-Kepler.md:27]: We may want to bake this in the conclusion. At the moment it's a bit separate (some information after the conclusion).

**Review by leonardpahlke — COMMENTED** [review]
Thanks @husky-parul for working on this blog short notice. I added a few minor comments. 🙌 

cc @mkorbi @catblade

**Comment by leonardpahlke** [comment]
> The preview does not have the link to this PR blog. Is this expected behavior? 

We need to add the header text to the blog (see comment https://github.com/cncf/tag-env-sustainability/pull/238#discussion_r1348024266). The blog will appear by the date we specify in the header.

File: website/content/blog/Embracing-Edge-Computing-with-Kepler.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Embracing Edge Computing with Kepler: A Dive into Remote Monitoring, Centralized Dashboarding, and Visualization
+
+In the ever-evolving landscape of technology, edge computing has emerged as a game-changer, enabling real-time data processing closer to the data source. Kepler, with its integration with OpenTelemetry (OTEL), has paved the way for exploring remote monitoring and centralized dashboarding options at the edge. This revolutionary approach offers invaluable insights into equipment and workload power consumption, making it a pivotal shift in the realm of energy observability.
+
+## Remote Monitoring Revolutionized
+
+Kepler metrics provide comprehensive insights into power consumption, empowering users with detailed analytics. The migration from Prometheus to OTLP exporter streamlines the process, eliminating the heavy Prometheus-based monitoring stack. Kepler’s OTLP exporters, operational on edge devices, facilitate centralized metric collection, ushering in a new era of efficient power and energy monitoring.
```
Comment (catblade) [inline website/content/blog/Embracing-Edge-Computing-with-Kepler.md:7]:
```suggestion
Kepler metrics provide comprehensive insights into power consumption, empowering users with detailed analytics. The migration from Prometheus to OTLP exporter streamlines the process by eliminating the heavy Prometheus-based monitoring stack. Kepler’s OTLP exporters which are operational on edge devices, facilitate centralized metric collection and usher in a new era of efficient power and energy monitoring.
```

File: website/content/blog/Embracing-Edge-Computing-with-Kepler.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+## The Power of Centralized Dashboarding
+
+Centralized dashboarding emerges as a cornerstone of this paradigm shift. With edge devices operational in various locations, having a centralized dashboard becomes indispensable. It offers a unified view of all the monitored parameters, ensuring seamless monitoring and identifying major power consuming workloads and/or devices. Centralized dashboarding ensures that insights into equipment and workload power consumption are readily accessible, enabling timely decision-making and proactive issue resolution.
```
Comment (catblade) [inline website/content/blog/Embracing-Edge-Computing-with-Kepler.md:11]:
```suggestion
Centralized dashboarding is a cornerstone of this paradigm shift. With edge devices operational in various locations, having a centralized dashboard becomes indispensable. It offers a unified view of all the monitored parameters, ensuring seamless monitoring and identifying major power consuming workloads and/or devices. Centralized dashboarding ensures that insights into equipment and workload power consumption are readily accessible, enabling timely decision-making and proactive issue resolution.
```

**Review by catblade — COMMENTED** [review]
I'd actually simplify some of this writing.  It will be really hard to access for a non-English speaker.

**Comment by husky-parul** [comment]
@leonardpahlke @catblade PTAL 
If you approve I can squash the commits.

Also @leonardpahlke 
In the preview I don't see the image, just the alternate text.

image

File: website/content/blog/Embracing-Edge-Computing-with-Kepler.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+---
+title: Embracing Edge Computing with Kepler - A Dive into Remote Monitoring, Centralized Dashboarding, and Visualization
+description: Kepler enables remote monitoring, centralized dashboarding, and visualization of energy consumption, revolutionizing energy observability at edge.
+slug: 2023-embracing-edge-computing-with-kepler
+weight: 1
+date:   2023-10-09 10:00:00 +0000
+author: Parul Singh
+---
+
```
Comment (leonardpahlke) [inline website/content/blog/Embracing-Edge-Computing-with-Kepler.md:9]:
```suggestion
*Community guest post as part of the [cloud native sustainability week](https://tag-env-sustainability.cncf.io/cloud-native-sustainability-week/)*
```
On the TAG ENV blog we need to highlight if a blog is from the TAG directly (an artifact from a WG for example)

Comment (leonardpahlke) [inline website/content/blog/Embracing-Edge-Computing-with-Kepler.md:9]:
```suggestion

 *Guest post as part of the [cloud native sustainability week](https://tag-env-sustainability.cncf.io/cloud-native-sustainability-week/)*
 ```

**Review by guidemetothemoon — COMMENTED** [review]
@husky-parul thank you for your contribution!
@leonardpahlke @catblade @husky-parul this blog post is due to be published today. @husky-parul are you able to go through the requested changes or can we merge those ourselves and perform final review?

**Comment by husky-parul** [comment]
@guidemetothemoon Please TAL

**Comment by guidemetothemoon** [comment]
Thank you for adding the changes, @husky-parul!
@leonardpahlke @catblade I have gone through your comments and resolved those according to the recent commits. Are we good to merge and publish this blog post?

**Comment by leonardpahlke** [comment]
Thanks @husky-parul 🚀

**Pull request merged**